project/plugins.sbt

```diff
@@ -3,3 +3,4 @@
 addSbtPlugin("com.eed3si9n"      % "sbt-buildinfo"    % "0.4.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-scalariform"  % "1.3.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-git"          % "0.8.4")
+addSbtPlugin("me.lessis"         % "bintray-sbt"      % "0.3.0")
```

Comment:
Yes, currently it's already possible to publish to a bintray account with this. But publishing to own account isn't that useful I suppose. What's missing is to have a ypg/mediative account as an organization, grant the right to the organization to people's account and then define the organization on the build file. At least that's my understanding, I haven't done it before.